src/core/core.controller.js

```diff
@@ -392,7 +392,7 @@ class Chart {
     });
   }
 
-  buildOrUpdateControllers() {
+  buildOrUpdateControllers(config) {
```

Comment:
This needs to be reflected in the typing:
https://github.com/chartjs/Chart.js/blob/0312697f38f88b3930f83e56d5850c4b4ff288c4/types/index.esm.d.ts#L512

Reply:
I removed this solution after I fixed the weird animation behavior bug.